app/helpers/application_helper/toolbar/chargeback_rates_center.rb

```diff
@@ -1,7 +1,7 @@
-class ApplicationHelper::Toolbar::ChargebacksCenter < ApplicationHelper::Toolbar::Basic
-  button_group('chargeback_vmdb', [
+class ApplicationHelper::Toolbar::ChargebackRatesCenter < ApplicationHelper::Toolbar::Basic
```

Comment:
Wouldn't this break the other 3 chargeback screens?

Reply:
@skateman this change is fine, we do not need toolbars for Chargeback Assignments and in CB saved Reports we only need one toolbar that is `chargeback_center.rb`